analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/LazyIngestionProvider.java

```diff
@@ -18,9 +18,14 @@
 
 package eu.fasten.analyzer.restapiplugin;
 
+
```

Comment:
Should this extra blank line be removed here?

Reply:
Remvoed on [28d46c8](https://github.com/fasten-project/fasten/pull/470/commits/28d46c85d7de232b6ede4bd34aa57fdaaa3e02da)